src/containers/Profile/index.js

```diff
@@ -81,16 +83,48 @@ export class Profile extends Component {
     this.setState({ [name]: value });
   };
 
+  isPhoneNumberValid = () => {
+    let phone = this.state.phoneNumber
+    let validNumbers = /\d+/g;
+    let updatedPhone = ""
+    let phoneNumberError = 'Please enter a valid 10 digit phone number with no dashes or parentheses.'
+      if (phone === "") {
+        return true;
+      }
+      else if (phone != null) {
+        if (phone.match(validNumbers) === null) {
+          this.setState({ phoneNumberError })
+          return false;
+        }
+        else if (phone.match(validNumbers) != null) {
+          updatedPhone = phone.match(validNumbers).join('')
+          if (updatedPhone.length !== 10) {
+            this.setState({ phoneNumberError });
+            return false;
+          } else {
+            this.setState({ phoneNumber: updatedPhone})
+            this.setState({ phoneNumberError: ' '});
```

Comment:
Is the space between the single quotes here intentional? I'm not sure what effect it would have, just double-checking